modules/installation-complete-user-infra.adoc

```diff
@@ -56,38 +56,38 @@ $ watch -n5 oc get clusteroperators
-cloud-credential                           4.8.0   True        False         False      29h
-cluster-autoscaler                         4.8.0   True        False         False      29h
-config-operator                            4.8.0   True        False         False      6h39m
-console                                    4.8.0   True        False         False      3h59m
-csi-snapshot-controller                    4.8.0   True        False         False      4h12m
-dns                                        4.8.0   True        False         False      4h15m
-etcd                                       4.8.0   True        False         False      29h
-image-registry                             4.8.0   True        False         False      3h59m
-ingress                                    4.8.0   True        False         False      4h30m
-insights                                   4.8.0   True        False         False      29h
-kube-apiserver                             4.8.0   True        False         False      29h
-kube-controller-manager                    4.8.0   True        False         False      29h
-kube-scheduler                             4.8.0   True        False         False      29h
-kube-storage-version-migrator              4.8.0   True        False         False      4h2m
-machine-api                                4.8.0   True        False         False      29h
-machine-approver                           4.8.0   True        False         False      6h34m
-machine-config                             4.8.0   True        False         False      3h56m
-marketplace                                4.8.0   True        False         False      4h2m
-monitoring                                 4.8.0   True        False         False      6h31m
-network                                    4.8.0   True        False         False      29h
-node-tuning                                4.8.0   True        False         False      4h30m
-openshift-apiserver                        4.8.0   True        False         False      3h56m
-openshift-controller-manager               4.8.0   True        False         False      4h36m
-openshift-samples                          4.8.0   True        False         False      4h30m
-operator-lifecycle-manager                 4.8.0   True        False         False      29h
-operator-lifecycle-manager-catalog         4.8.0   True        False         False      29h
-operator-lifecycle-manager-packageserver   4.8.0   True        False         False      3h59m
-service-ca                                 4.8.0   True        False         False      29h
-storage                                    4.8.0   True        False         False      4h30m
+NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+authentication                             4.8.2     True        False         False      19m
```

Comment:
@ahardin-rh Do you happen to know whether anyone went through and did a 4.8.0->4.8.2 sweep once we found out that it wasn't being released as 4.8.0? Or is that something that we still need to do?